setup.py

```diff
@@ -37,11 +37,11 @@
     'packages': find_packages(where="."),
     'entry_points': {
         'pennylane.plugins': [
-            'default.qubit = pennylane.plugins:DefaultQubit',
-            'default.gaussian = pennylane.plugins:DefaultGaussian',
-            'default.qubit.tf = pennylane.plugins.default_qubit_tf:DefaultQubitTF',
-            'default.tensor = pennylane.beta.plugins.default_tensor:DefaultTensor',
-            'default.tensor.tf = pennylane.beta.plugins.default_tensor_tf:DefaultTensorTF',
+            'default.qubit = pennylane.devices:DefaultQubit',
+            'default.gaussian = pennylane.devices:DefaultGaussian',
+            'default.qubit.tf = pennylane.devices.default_qubit_tf:DefaultQubitTF',
+            'default.tensor = pennylane.beta.devices.default_tensor:DefaultTensor',
+            'default.tensor.tf = pennylane.beta.devices.default_tensor_tf:DefaultTensorTF',
```

Comment:
This is somehow not working...

Reply:
@josh146

Reply:
I see, one needs to redo ``pip install -e .``

Reply:
@mariaschuld sorry I missed this comment! Does it work now?

Reply:
Yes!